dotnet/Dockerfile

```
@@ -5,5 +5,7 @@ USER root
 RUN wget -q https://packages.microsoft.com/config/ubuntu/19.04/packages-microsoft-prod.deb && \
     dpkg -i packages-microsoft-prod.deb && rm -rf packages-microsoft-prod.deb && \
     add-apt-repository universe && \
-    apt-get update && apt-get -y -o APT::Install-Suggests="true" install dotnet-sdk-3.1 && \
-    rm -rf /var/lib/apt/lists/*;
+    apt-get update && \
+    apt-get -y -o APT::Install-Suggests="true" install dotnet-sdk-3.1 && \
+    apt-get install -y fsharp mono-complete && \
```

Let's try combining the two `apt-get install`s:

```suggestion
    apt-get -y -o APT::Install-Suggests="true" install \
        dotnet-sdk-3.1 \
        fsharp \
        mono-complete && \
```